src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -0,0 +1,337 @@
+
+### FAQ [#faq]
+
+<CollapserGroup>
+<Collapser
+    id="user-data-collection"
+    title="How is user data collected while maintaining privacy?"
+>
+    * Minimal data storage: Session replay does not store screenshots or videos. It only records the essential DOM state changes required to recreate the user's experience. This results in a much smaller amount of data being stored.
+    * Client-side privacy enforcement: Privacy settings are applied directly on the client-side, ensuring strong protection of user data from the outset.
+    * Robust default privacy settings: Session replay includes default privacy settings designed to limit the potential sharing of sensitive customer information. These settings can be further customized to meet specific privacy requirements.
+</Collapser>
+
+<Collapser
+    id="performance-impact"
+    title="What's the impact on app performance?"
+>
+    * Minimal initial impact: Session replay loads asynchronously after full page load, ensuring no interference with early page load performance metrics.
+* DOM mutation tracking: Once loaded, session replay monitors DOM changes and reports them as compressed payloads. The size of these payloads correlates with the complexity of the DOM tree.
+* Performance optimization techniques: Payload size can also increase if the DOM is large. However, session replay uses a number of techniques to reduce payload size, such as:
+    * GZIP compression
+    * Duplication reduction
+    * Strategic payload harvesting: It only sends a large (compressed) payload when the full DOM is captured, which typically only happens when the replay feature is first imported. After that, it will only send small payloads that relate to the end-user's actions.
+</Collapser>
+
+<Collapser
+    id="data-retention-limits"
+    title="What are the data retention limits?"
+>
+    Replays have a standard retention period of 8 days.
+</Collapser>
+
+</CollapserGroup>
+
+
+## Get started [#get-started]
+
+<Steps>
+
+  <Step>
+    ### Enable and configure session replay [#enable-configure-settings]
+
+    1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > All Capabilities > Browser**.
+    2. Select your browser app.
+    3. In the left-hand menu, click  **Application settings**.
+    2. On the Application settings page, make sure **Pro** or **Pro + SPA** browser agent is selected. Session replay is not available for the Lite browser agent.
+    3. Toggle ON **Session tracking** and **Session replay** settings.
+    </Step>
+
+  <Step>
+    ### Configure session sampling rates [#configure-sampling-rates]
+    
+    Make selections for the following sampling rates:
```

Comment:
```suggestion
Set a sampling rate (0%-100%) for the following samples:
```

Reply:
I wasn't entirely sure what the action in this step was at first. You don't have to use the above suggestion, but I'd clarify that you ahve to set a percentage for EACH of the bullets below - at first I assumed I was choosing which sampling type to use.